doc/api/cli.md

```diff
@@ -1229,6 +1229,16 @@ Starts the Node.js command line test runner. This flag cannot be combined with
 See the documentation on [running tests from the command line][]
 for more details.
 
+### `--test-coverage`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+When used in conjunction with the `node:test` module, a code coverage report is
```

Comment:
What happens when used without `node:test`? it's a noop I assume?

Reply:
Yes, I'll elaborate that part a bit. This flag puts the coverage responsibilities in the hands of the test runner.